server/server.js

```diff
@@ -95,7 +95,7 @@
     var twitter = new Twitter(conf);
     //pull in 100 tweets excluding rts and replies
     var error = function (err, response, body) {
-        console.log('ERROR [%s]', err);
+        res.status(404).send('Not found');
```

Comment:
'Another common failure is when the client application asks for some non existent resource. In that case a404 Not Found status is returned to the client.'

https://www.develop.com/httpstatuscodesrest